src/main/java/javapoz24/team3/rental/domain/rental/Address.java

```diff
@@ -0,0 +1,22 @@
+package javapoz24.team3.rental.domain.rental;
+
+import javapoz24.team3.rental.domain.base.BaseEntity;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.OneToOne;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Address extends BaseEntity {
```

Comment:
Tutaj raczej zrobiłbym Embaded niż zwykła encja, ze względu na to że adres często będzie wykorzystywany w innych encjach.
Druga opcja to jednostronna relacja CompanyBranch -> Addres. Adress nie powinien wiedzieć nic na temat CompanyBranch. Jak pojawią się kolejne encje korzystające z adresu to raczej nie chcesz ich tutaj dodawać.

Reply:
branch dev zrobiony, poprawię jeszcze nieco ten kod i zrobię PR już na deva (a tego PR się pewnie odrzuci).

Zrobiłem Embeddable/Embedded dla klasy Adress - za dużo połączeń na początek... i tak teraz niewiele działa.